### PR TITLE
Fix/sniff soc space management

### PR DESCRIPTION
**Pull request by ilyeskallala**

### CODE REVIEW

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCSpaceManagementTestIT.java [hunk 1]
```diff
@@ -0,0 +1,599 @@
...
+   * <li>Pre-Condition:</li>
+   * <li>Post-Condition:</li> Step Number: 1 Step Name: Step 1: Check hidden space
+   * Step Description: - Login as user1 - Add new hidden space - Logout - Login as
+   * user2 - access the url of an hidden space that she is not member of via url :
+   * /portal/g/:spaces:hidden/hidden/forum/ - Click on [Find Spaces] link Input
+   * Data: Expected Outcome: - Add new space successfully - A page with 'Space not
+   * Found' title is displayed Message is : No space is available at this URL.
+   * [Find Spaces] - User2 is redirected to Spaces directory page
+   */
+  @BugInPLF("SOC-5884")
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCSpaceManagementTestIT.java]: The issue SOC-5884 is closed as Normal Behavior. Should we remove this tag (and maybe update the test) ?

File: platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCSpaceManagementTestIT.java [hunk 2]
```diff
@@ -0,0 +1,599 @@
...
+    spaceManagement.addNewSpaceSimple(space, space);
+    info("Set permission for the space");
+    spaceHomePage.goToSpaceSettingTab();
+    spaceSettingManagement.goToAccessEditTab();
+    spaceSettingManagement.setPermissionForSpace(arrayRight);
+    spaceManagement.goToActivityStreamTab();
+    refresh();
+    String urlSpace = url();
+    manageLogInOut.signIn(username2, password);
+    for (int i = 0; i <= 14; i++) {
```
Comment (thomasdelhomenie) [inline platform/src/test/java/org/exoplatform/platform/qa/ui/platform/social/SOCSpaceManagementTestIT.java]: Magic number :) https://en.wikipedia.org/wiki/Magic_number_(programming)#Unnamed_numerical_constants

File: selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceManagement.java [hunk 3]
```diff
@@ -176,29 +177,28 @@ public void goToAccessTabFromPopUp() {
...
       info("evt.click on change picture button");
-      evt.click(ELEMENT_SPACE_CHANGE_AVATAR_BTN);
-      info("evt.click on upload button");
-      evt.click(ELEMENT_UPLOAD_POPUP_SELECT_FILE_BTN);
+      $(ELEMENT_SPACE_CHANGE_AVATAR_BTN).click();
+     $(byClassName("uploadContainer")).find(byClassName("file")).uploadFromClasspath(filepath);
 
       info("filepath:" + filepath);
-      testBase.uploadFileUsingRobot(filepath);
+      //testBase.uploadFileUsingRobot(filepath);
```
Comment (thomasdelhomenie) [inline selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceManagement.java]: To be removed if not needed anymore.

Comment (thomasdelhomenie) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceManagement.java]: @ilyeskallala comment not considered.

Comment (ilyeskallala) [reply selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/platform/social/SpaceManagement.java]: line removed now

**Pull request merged**